docs/src/main/paradox/quickstart-scala/index.md

```diff
@@ -0,0 +1,204 @@
+Akka HTTP Quickstart for Scala
+==============================
+
+Akka HTTP offers a general toolkit for providing and consuming HTTP-based services. The Akka HTTP modules implement a full server- and client-side HTTP stack on top of `akka-actor` and `akka-stream`. A typical application does not sit on top of Akka HTTP. Instead, Akka HTTP makes it easier to build integration layers based on HTTP, and therefore stays on the sidelines. This allows you to base your app on what makes sense and use Akka HTTP for HTTP integration.
+
+Akka HTTP is not a framework--not because we don't like frameworks--but to provide maximum flexibility. For example, you might use the Play framework to implement browser-based interactions or Lagom framework for creating microservices, both of them are also based on Akka.
+
+Akka HTTP follows a rather open design and often offers several APIs for accomplishing the same thing. You can choose the API with the level of abstraction that best suits your application.  If you have trouble achieving something using a high-level API, you can probably find a lower-level API to use. The low-level APIs offer more flexibility, but might require you to write more application code.
+
+This guide highlights Akka HTTP fundamentals in the context of a simple user registry [REST service](https://en.wikipedia.org/wiki/Representational_state_transfer). Within 30 minutes, you should be able to:
+
+* Download the quickstart project and run the app.
+* Follow this guide to explore the code.
+* Integrate the project into IntelliJ. If you prefer a different IDE, the principles should be similar, as long as it supports Scala.
+
+## Prerequisite
+
+Having a basic understanding of Akka actors will make it easier to understand the example.
+If you are new to Akka, we recommend completing the
+@extref[Akka Quickstart](akka-docs:typed/actors.html#first-example) before this tutorial.
+
+You can run the Akka HTTP example project on Linux, MacOS, or Windows. The only prerequisite is Java 17 or later.
+
+## Downloading the example
+
+The Akka HTTP example for Scala is a zipped project that includes a distribution of the [sbt](http://www.scala-sbt.org) build tool. Download and unzip the example as follows:
+
+1. Download the project [zip file](../attachments/akka-http-quickstart-scala.zip).
+2. Extract the zip file to a convenient location.
+
+* On Linux and MacOS systems, open a terminal and use the command `unzip` akka-quickstart-scala.zip. Note: On OSX, if you unzip using Archiver, you also have to make the sbt files executable by entering the following two commands:
+
+```
+$ chmod u+x ./sbt
+$ chmod u+x ./sbt-dist/bin/sbt
```

Comment:
We don't really have those for sbt included. I think we should require a local installation of sbt.